lib/scan/admin/engine.py

```diff
@@ -90,16 +91,23 @@ def check(target, user_agent, timeout_sec, log_in_file, language, time_sleep, th
                                'PORT': "", 'TYPE': 'admin_scan',
                                'DESCRIPTION': messages(language, "found").format(target, r.status_code, r.reason),
                                'TIME': now(), 'CATEGORY': "scan", 'SCAN_ID': scan_id, 'SCAN_CMD': scan_cmd}, language, thread_tmp_filename)
-            if r.status_code is 200:
-                for dlmsg in directory_listing_msgs:
-                    if dlmsg in content:
-                        info(messages(language, "directory_listing").format(target), log_in_file, "a"
-                                           ,{'HOST': target_to_host(target), 'USERNAME': '', 'PASSWORD': '',
-                                           'PORT': "", 'TYPE': 'admin_scan',
-                                           'DESCRIPTION': messages(language, "directoy_listing").format(target), 'TIME': now(),
-                                           'CATEGORY': "scan", 'SCAN_ID': scan_id, 'SCAN_CMD': scan_cmd}, language, thread_tmp_filename)
-                        __log_into_file(log_in_file, 'a', data, language)
-                        break
+        if r.status_code is 200:
```

Comment:
"is" needs to be "==" here and in other places:
flake8:
lib/scan/admin/engine.py:84:12: F632 use ==/!= to compare constant literals (str, bytes, int, float, tuple)
lib/scan/admin/engine.py:94:12: F632 use ==/!= to compare constant literals (str, bytes, int, float, tuple)
lib/scan/admin/engine.py:202:12: F632 use ==/!= to compare constant literals (str, bytes, int, float, tuple)
lib/scan/admin/engine.py:240:35: F632 use ==/!= to compare constant literals (str, bytes, int, float, tuple)
lib/scan/admin/engine.py:245:20: F632 use ==/!= to compare constant literals (str, bytes, int, float, tuple)
lib/scan/admin/engine.py:250:12: F632 use ==/!= to compare constant literals (str, bytes, int, float, tuple)
lib/scan/admin/engine.py:251:16: F632 use ==/!= to compare constant literals (str, bytes, int, float, tuple)